docs/coding-conventions.md

```diff
@@ -320,6 +320,45 @@ coding conventions](#detailed-coding-conventions).
       subdirectories.
 - Document directories and filenames should use dashes rather than underscores.
 
+## Functional tests conventions
+
+- Avoid using constants imported from product code package. Instead, define the expected
+  constant values in the test suite. This refers to any aspect of the test suite.
```

Comment:
would you add the reason for this? I suppose the point here is to protect against inadvertent change of the constant.

Reply:
+1

I am not really convinced this convention is correct in general. It may be correct for specifics.
E.g. formal API constants can be used while informal ones (from inner packages) cannot.

Protecting the name of constants is not needed which the value can be protected by unit tests, focusing on protecting changes for backward compatibility.

In any case, detailed reasoning why such a convention should be places must be given.